### ISSUE TITLE
Reading aloud incoherently with input stream and eleven_multilingual_v1

### ISSUE THREAD

**Issue by themw123** [elevenlabs/elevenlabs-python#78]
It seems that input stream is stuttering with eleven_multilingual_v1. Hope it will get fixed, because input stream is a great feature.

**Comment by aavetis** [comment]
think i'm seeing the same thing, here's a recording. at certain points of the stream the voice will cut out. it feels like either the connection dropping, or the stream interrupting itself. can't tell.

https://github.com/elevenlabs/elevenlabs-python/assets/3076502/622fda16-df4e-46ae-abd1-ded13604f210

**Comment by aavetis** [comment]
After further investigation, this seems to be an issue when using `play` to play the individual chunks vs using  `stream`.

For example, this works well, right from the docs, but doesn't use streaming input. Only streaming output.
```py
from elevenlabs import generate, stream

audio_stream = generate(
  text="This is a... streaming voice!!",
  stream=True
)

stream(audio_stream)

**Comment by themw123** [comment]
i tested it again and it occurs with almost all male voices. @aavetis as i said the following settings cause the error: input stream, play with stream() and eleven_multilingual_v1

**Comment by flavioschneider** [comment]
It's expected that using the `play` function on individual chunks doesn't have a smooth stream (that's why a separate `stream` function is provided). Other than that, the interruptions might happen when you don't yield chunks fast enough and the input stream stalls.

**Comment by themw123** [comment]
@flavioschneider 
Yes i am using the stream function. As i said, i used the the following:  input stream, stream() and eleven_multilingual_v1. But today i tested it again and the problem seems to be solved. I changed nothing, so i think the problem was related to eleven_multilingual_v1. and you guys fixed it.

**Comment by themw123** [comment]
@flavioschneider
Yes i am using the stream function. As i said, i used the the following: input stream, stream() and eleven_multilingual_v1. But today i tested it again and the problem seems to be solved. I changed nothing, so i think the problem was related to eleven_multilingual_v1. and you guys fixed it.

**Comment by themw123** [comment]
ohhh

**Comment by flavioschneider** [comment]
Perfect!
